packages/styles/src/components/form-textarea.scss

```diff
@@ -1,17 +1,188 @@
-@use './../variables/spacing';
+@forward './../variables/options';
+
+@use './../mixins/utilities';
 @use './../variables/components/forms';
-@use './../functions/sizing';
+@use './../mixins/forms' as forms-mx;
+
+@use './../functions/tokens';
+@use './../tokens/components';
+@use './../tokens/elements';
+
+tokens.$default-map: components.$post-text-area;
 
 textarea.form-control {
-  &::-webkit-resizer {
-    background-image: url("data:image/svg+xml,%3Csvg viewBox='0 0 15 15' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath fill='#{forms.$input-border-color}' d='M0 12H3V15H0V12Z'/%3E%3Cpath fill='#{forms.$input-border-color}' d='M6 12H9V15H6V12Z'/%3E%3Cpath fill='#{forms.$input-border-color}' d='M6 6H9V9H6V6Z'/%3E%3Cpath fill='#{forms.$input-border-color}' d='M12 6H15V9H12V6Z'/%3E%3Cpath fill='#{forms.$input-border-color}' d='M12 0H15V3H12V0Z'/%3E%3Cpath fill='#{forms.$input-border-color}' d='M12 12H15V15H12V12Z'/%3E%3C/svg%3E");
-    background-repeat: no-repeat;
-    background-size: 75%;
-  }
+  display: block;
+  width: 100%;
+  font-family: tokens.get('body-font-family', elements.$post-body);
```

Comment:
Isn't it inherited?

Reply:
the font-family is, I removed it but the other two aren't (display is inline-block and width is not 100% of its container)